ref/general/packaging-deploying.adoc

```diff
@@ -14,79 +14,69 @@
 link:https://maven.apache.org/[Maven] is a build automation tool that makes managing your builds and dependencies easier.
 With Maven, you use a Project Object Model (POM) file written in XML to define information about your project, and Maven downloads and installs all dependencies for you.
 The link:https://github.com/OpenLiberty/ci.maven[Open Liberty Maven plug-in] helps manage your Open Liberty server and applications.
-
-After you enable the Maven plug-in, specify `war` in the `<packaging>` element in your `pom.xml` file.
-Then, run the `mvn package` command to package your application and Open Liberty into a WAR file.
-This command compiles, verifies, and builds the project.
-After you create this server package, you're ready to containerize and deploy it.
 To learn more about the Open Liberty Maven plug-in, check out this guide on link:/guides/maven-intro.html[Building a web application with Maven].
 
-=== Gradle
 Like Maven, link:https://gradle.org/[Gradle] is a build tool that provides dependency management for your projects.
 However, Gradle has it's own domain-specific language (DSL) that's based on Groovy.
 Similar to the Maven plug-in, the link:https://github.com/OpenLiberty/ci.gradle[Open Liberty Gradle plug-in] provides tasks for you to manage your server and applications.
-
-After you install the Gradle plug-in, include the following lines of code at the top of your `build.gradle` file to specify required plug-ins:
-----
-apply plugin: 'war'
-apply plugin: 'liberty'
-----
-Then, run the link:https://github.com/OpenLiberty/ci.gradle/blob/master/docs/libertyPackage.md#libertypackage-task[`libertyPackage` task] to package your application and Open Liberty into a WAR file.
-Once you create this server package, you can containerize and deploy it.
 To learn more about the Open Liberty Gradle plug-in, check out this guide on link:https://openliberty.io/guides/gradle-intro.html[Building a web application with Gradle].
 
-=== Runnable JAR file
-You can also use Maven or Gradle to package your application with Open Liberty into a runnable JAR file that contains the application and server runtime.
+== Server packages
+Open Liberty supports different server package types.
+The supported package types are `zip`, `jar`, `tar`, and `tar.gz`.
+If you don't specify a package type, then `zip` is the default, meaning that your application and an Open Liberty installation are packaged into a ZIP file.
+
+For more information about Open Liberty packaging options with Maven, see the link:https://github.com/OpenLiberty/ci.maven/blob/master/docs/package.md#package[`package` goal in the Maven documentation]. For more information about Open Liberty packaging options with Gradle, see the link:https://github.com/OpenLiberty/ci.gradle/blob/master/docs/libertyPackage.md#libertypackage-task[`libertyPackage` task in the Gradle documentation].
+
+=== Packaging for containerization
+Use Maven or Gradle to package an application before you containerize it.
```

Comment:
This use of package is different from the prior sections use of package as a term. In this case you are talking about creating a war and ear, in the prior one you were talking about a server package. I think this should probably exist prior to talking about server packages.